tox -e codestyle fails on astropy/_erfa/core.py

<!-- This comments are hidden when you submit the issue,
so you do not need to remove them! -->

<!-- Please be sure to check out our contributing guidelines,
https://github.com/astropy/astropy/blob/master/CONTRIBUTING.md .
Please be sure to check out our code of conduct,
https://github.com/astropy/astropy/blob/master/CODE_OF_CONDUCT.md . -->

<!-- Please have a search on our GitHub repository to see if a similar
issue has already been posted.
If a similar issue is closed, have a quick look to see if you are satisfied
by the resolution.
If not please go ahead and open an issue! -->

<!-- Please check that the development version still produces the same bug.
You can install development version with
pip install git+https://github.com/astropy/astropy
command. -->

### Description
<!-- Provide a general description of the bug. -->

`tox -e codestyle` always fails with a huge stream of code style violations in `astropy/_erfa/core.py`. This renders it largely useless for checking my own code for style violations.

I assume the file is auto-generated, so simply telling the job to ignore code style violations in that file might solve the problem. If not, a once-over with `black` or `autopep8` would probably resolve the issue.

### Expected behavior
<!-- What did you expect to happen. -->

`tox -e codestyle` succeeds on a clean master.

### Actual behavior
<!-- What actually happened. -->
<!-- Was the output confusing or poorly described? -->
```
...
astropy/_erfa/core.py:20058:37: W291 trailing whitespace
astropy/_erfa/core.py:20101:37: W291 trailing whitespace
astropy/_erfa/core.py:20146:37: W291 trailing whitespace
astropy/_erfa/core.py:20187:37: W291 trailing whitespace
astropy/_erfa/core.py:20231:37: W291 trailing whitespace
astropy/_erfa/core.py:20272:37: W291 trailing whitespace
astropy/_erfa/core.py:20342:22: W292 no newline at end of file
244
ERROR: InvocationError for command /home/archibald/projects/astropy/astropy/.tox/codestyle/bin/flake8 astropy --count --select=E101,W191,W291,W292,W293,W391,E111,E112,E113,E30,E502,E722,E901,E902,E999,F822,F823 (exited with code 1)
_______________________________________________________ summary _______________________________________________________
ERROR:   codestyle: commands failed
```

### Steps to Reproduce
<!-- Ideally a code example could be provided so we can run it ourselves. -->
<!-- If you are pasting code, use triple backticks (```) around
your code snippet. -->
<!-- If necessary, sanitize your screen output to be pasted so you do not
reveal secrets like tokens and passwords. -->

```sh
tox -e codestyle
```

### System Details
<!-- Even if you do not think this is necessary, it is useful information for the maintainers.
Please run the following snippet and paste the output below:
import platform; print(platform.platform())
import sys; print("Python", sys.version)
import numpy; print("Numpy", numpy.__version__)
import astropy; print("astropy", astropy.__version__)
import scipy; print("Scipy", scipy.__version__)
import matplotlib; print("Matplotlib", matplotlib.__version__)
-->
```
>>> import platform; print(platform.platform())
Linux-5.4.0-33-generic-x86_64-with-glibc2.29
>>> import sys; print("Python", sys.version)
Python 3.8.2 (default, Apr 27 2020, 15:53:34) 
[GCC 9.3.0]
>>> import numpy; print("Numpy", numpy.__version__)
Numpy 1.18.4
>>> import astropy; print("astropy", astropy.__version__)
astropy 4.2.dev162+gd0a1e469f
>>> import scipy; print("Scipy", scipy.__version__)
Scipy 1.4.1
>>> import matplotlib; print("Matplotlib", matplotlib.__version__)
Matplotlib 3.2.1
```

--- a/setup.cfg
+++ b/setup.cfg
@@ -125,11 +125,12 @@ bitmap = static/wininst_background.bmp
 
 [flake8]
 max-line-length = 100
-exclude = extern,*parsetab.py,*lextab.py
+exclude = extern,*parsetab.py,*lextab.py,astropy/_erfa/core.py
+
 
 [pycodestyle]
 max-line-length = 100
-exclude = extern,*parsetab.py,*lextab.py
+exclude = extern,*parsetab.py,*lextab.py,astropy/_erfa/core.py
 
 [isort]
 line_length = 99

